src/Store.java

```diff
@@ -1,27 +1,39 @@
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.Scanner;
 
 public class Store {
 
     public static void main(String[] args) {
-        ArrayList<String> inventory = new ArrayList<>();
-        inventory.add("Tastatura XZ");
-        inventory.add("Tastatura YY");
-        inventory.add("Televizor LG KY");
-        inventory.add("Televizor Samsung X1");
-        inventory.add("Ves masina 12");
-        inventory.add("Ves masina X4");
-        inventory.add("Pegla TT");
+        //Za svaki proizvod ukljuci i njegovu cenu i kolicinu
+
+        ArrayList<HashMap<String, String>> inventory = new ArrayList<>();
+
+        inventory.add(createProduct("Tastatura XZ", "3000", "3"));
```

Comment:
Maybe this should be a separate function that is focused on creating the inventory.